RELEASE-NOTES.txt

```diff
@@ -1,6 +1,7 @@
 12.7
 -----
 * Block Editor: Video, Quote and More blocks are available now.
+* Offline Improvements: Posts will now be auto-uploaded when reopening the App or when a connection is detected.
```

Comment:
Should we say, failed to upload posts...? or something like "Post that failed to upload due to connectivity issues will be auto-uploaded..."?

Reply:
Changed, and also fixed a typo in another item.